src/utils/html.js

```diff
@@ -387,6 +387,7 @@ export function decodeCharacterReferences(html) {
 const lessThan = /</g;
 const greaterThan = />/g;
 const amp = /&/g;
+const quote = /"/g;
```

Comment:
Hmm, it somehow worked correctly without this before, why is it needed now?

Reply:
I think it used this before: https://github.com/ractivejs/ractive/blob/dev/src/utils/dom.js#L154-L159 (which escapes the correct chars for attributes)

Reply:
Ah! I was wondering that too, but I didn't look around hard enough. Thanks!